pkg/cluster/describe.go

```diff
@@ -272,3 +274,37 @@ func PrintClusterDescription(connection *sdk.Connection, cluster *cmv1.Cluster)
 
 	return nil
 }
+
+// findHyperShiftMgmtSvcClusters returns the name of a HyperShift cluster's management and service clusters.
+// It essentially ignores error as these endpoint is behind specific permissions by returning empty strings when any
+// errors are encountered, which results in them not being printed in the output.
+func findHyperShiftMgmtSvcClusters(conn *sdk.Connection, cluster *cmv1.Cluster) (string, string) {
+	if !cluster.Hypershift().Enabled() {
+		return "", ""
+	}
+
+	hypershiftResp, err := conn.ClustersMgmt().V1().Clusters().
+		Cluster(cluster.ID()).
+		Hypershift().
+		Get().
+		Send()
+	if err != nil {
+		return "", ""
+	}
+
+	mgmtClusterName := hypershiftResp.Body().ManagementCluster()
```

Comment:
what will be the output of this command for users who can't access the management cluster details?